rosdep/python.yaml

```diff
@@ -8636,6 +8636,12 @@ python3-pylibdmtx:
     focal:
       pip:
         packages: [pylibdmtx]
+python3-pymap3d:
```

Comment:
If we think this will be released into future distributions, we can instead use the wildcard to set the package for all variants and specify `null` for the specific distributions with active ROS distributions which don't have it (e.g. 20.04). That lets this work with the next distribution without later needing to modify again